scripts/local/launch/helpers/create-azure-resources.cmd

```diff
@@ -3,6 +3,10 @@
 :: For more info on pcs-cli visit (https://github.com/Azure/pcs-cli).
 @echo off
 
+:: strlen of /scripts/local/launch/helpers/ => 30
+SET APP_HOME=%~dp0
+SET APP_HOME=%APP_HOME:~0,-30%
```

Comment:
You will have to change this if the folder path or name changes since it is hard coded to length. Isn't there a better way?

Reply:
I tried this with unix style code. It doesn't seem to work.